apis/Google.Longrunning/Google.Longrunning/PollSettings.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+
+namespace Google.Longrunning
+{
+    // TODO: Should this actually be in GAX? Could be useful for other situations,
+    // although potentially those situations *should* use LRO...
+
+    // TODO: Rename to RepeatedPollSettings or something similar?
+
+    /// <summary>
+    /// Settings controlling repeated polling until an operation completes.
+    /// </summary>
+    public class PollSettings
```

Comment:
Hmm, true. Will think about that for a follow-up PR. (I have other things I want to do anyway.)
Given that CallSettings is mutable, this is already not properly immutable...